src/foundry/client/apps/sidebar/tab-directory.d.ts

```diff
@@ -94,27 +105,16 @@ declare global {
       }
     ): [T['folders'], T['documents']];
 
-    /**
-     * @override
-     */
-    render(force?: boolean, context?: SidebarDirectory.RenderContext<Options>): this | void;
+    override render(force?: boolean, context?: SidebarDirectory.RenderContext<Options>): this | void;
 
-    /**
-     * @param options - (unused)
-     * @override
-     * @remarks
-     * This implementation doesn't actually return `PlaylistDirectory.Data`, it's only to allow {@link PlaylistDirectory}
-     * to override accordingly.
```

Comment:
Ditto.

Reply:
Intentional, if you take a look at the return type, it doesn’t mention PlaylistDirectory.Data anymore. This has been cleaned up in the mean time, and the remark is obsolete.